doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+
+Plutus tools component descriptions
+=================================================
+
+The Plutus team has provided a set of “Tools”---components, libraries and 
+executables---for external developers to use. They are located in the 
+`plutus-apps <https://github.com/input-output-hk/plutus-apps>`_ repository. 
+
+For each tool listed below, we have indicated its specific location within 
+plutus-apps, provided a short description, indicated its dependencies, and given 
+a brief statement about its state of development to give you some context. 
+
+1. Plutus script utils
+--------------------------
+
+`Plutus script utils <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-script-utils>`_ 
+is a utility library for helping users write Plutus scripts that are to be used on-chain. 
+Currently, this is a “kitchen sink” library containing code that was moved out of 
+plutus-ledger-api. 
+
+It provides: 
+
+   * hashing functions for Datums, Redeemers and Plutus scripts for any Plutus language version. 
+   * functionality for wrapping the untyped Plutus script with a typed interface. 
+   * utility functions for working with the ScriptContext of a Plutus Script. 
+
++--------------+---------------------------------------------------------------+
+| Plutus Script Utils                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-script-utils``                           |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api                                |
++--------------+---------------------------------------------------------------+
+| State        | Work in progress with an unstable interface.                  |
++--------------+---------------------------------------------------------------+
+
+2. Marconi
+--------------------------
+
+`Marconi <https://github.com/input-output-hk/plutus-apps/tree/main/marconi>`_ is an 
+application for indexing useful data from the Cardano blockchain in a scalable way. 
+
++--------------+---------------------------------------------------------------+
+| Marconi                                                                      |
++==============+===============================================================+
+| Location     | ``plutus-apps/marconi``                                       |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api                                |
++--------------+---------------------------------------------------------------+
+| State        | In a very early alpha version. Has only a few indexers and    |
+|              | the final architecture is currently being designed.           |
++--------------+---------------------------------------------------------------+
```

Comment:
I think the architecture is close to final, maybe we want to say its just not fully implemented yet? What do you think @raduom

Reply:
It appears that the description we have now for State is essentially accurate. But maybe we can use language that has a more affirming tone. For example: 

"State: Currently in an early alpha version. Has some indexers. Some aspects of the architecture are being designed."